pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -94,6 +73,25 @@ func (r *RegistryFlags) AsRegistryOpts() registry.Opts {
 	if os.Getenv("IMGPKG_ANON") == "true" {
 		opts.Anon = true
 	}
+	iaasAuth, found := os.LookupEnv("IMGPKG_ENABLE_IAAS_AUTH")
+	if found {
+		if iaasAuth == "true" {
+			opts.EnableIaasAuthProviders = true
+		}
+	}
+
+	keychains, found := os.LookupEnv("IMGPKG_ACTIVE_KEYCHAINS")
+	if found {
+		if len(keychains) > 0 {
```

Comment:
nit: Any opposition to using `&&` to reduce the amount of branching going on?

Reply:
No reason, changed to &&